### ISSUE TITLE
Parallel NPM Commands on Windows

### ISSUE THREAD

**Issue by Wolfeitz** [jmfurlott/react-isomorphic-boilerplate#3]
NPM on Windows can not string commands in parallel such as this - "start": "npm run watch-js & npm run dev-server & npm run server"
To solve this I added npm-run-all (https://www.npmjs.com/package/npm-run-all) and changed the start command in package.json to this "start": "npm-run-all --parallel watch-js dev-server server".

Works like a champ!!!

**Comment by jmfurlott** [comment]
Hmm, I'm sorry but I never even considered Windows.  @Wolfeitz would you mind adding some Windows instructions to the README as a PR?

Also, are you using cygwin or anything?

**Comment by Wolfeitz** [comment]
Nope, not running anything like that.  Only pre-reqs are Node and Git.
